test/functional/test_framework/util.py

```diff
@@ -569,3 +569,14 @@ def mine_large_block(node, utxos=None):
     fee = 100 * node.getnetworkinfo()["relayfee"]
     create_lots_of_big_transactions(node, txouts, utxos, num, fee=fee)
     node.generate(1)
+
+def find_vout_for_address(node, txid, addr):
+    """
+    Locate the vout index of the given transaction sending to the
+    given address. Raises runtime error exception if not found.
+    """
+    tx = node.getrawtransaction(txid, True)
+    for i in range(len(tx["vout"])):
```

Comment:
Alternatively:

```python
    for i, vout in enumerate(tx["vout"]):
        if any([addr == a for a in vout["scriptPubKey"]["addresses"]]):
            return i
```

Reply:
Thanks! It doesn't feel strictly necessary but it's marginally better. Will do if this needs a rebase or gets more requests for other changes.

Reply:
Agreed, no need to change the branch for just this change